src/sections/shared/add-data-actions/AddDataActionsButton.tsx

```diff
@@ -21,6 +21,9 @@ export default function AddDataActionsButton({
 
   const createCollectionRoute = RouteWithParams.CREATE_COLLECTION(collectionId)
   const createDatasetRoute = RouteWithParams.CREATE_DATASET(collectionId)
+  // TODO: Enable/disable review creation based on config setting. See also LoggedInHeaderActions.tsx
+  // TODO: Longer term, query /api/datasets/datasetTypes and show those
```

Comment:
I created this PR to query /api/datasets/datasetTypes

- https://github.com/IQSS/dataverse-client-javascript/pull/364